_15thnight/util.py

```diff
@@ -0,0 +1,44 @@
+import simplejson as json
+from flask import current_app
+from flask.json import dumps
+from flask.ext.login import current_user
+from functools import wraps
+
+
+class ExtensibleJSONEncoder(json.JSONEncoder):
+    """A JSON encoder that returns the to_json method if present"""
+    def default(self, obj):
+        if hasattr(obj, 'to_json'):
+            return obj.to_json()
+        return super(ExtensibleJSONEncoder, self).default(obj)
+
+def jsonify(*args, **kwargs):
```

Comment:
I guess the different way you like to handle jsonify could work. Seems a bit wonky to me I suppose. However, flask's jsonify returns a Response object as well, which allows you to change the status code too:

    jsonified_response.status_code = 200

Your jsonify can work for now.